src/lib/cyleaflet_clientside.js

```diff
@@ -0,0 +1,88 @@
+import proj4 from 'proj4'
```

Comment:
I'm a little concerned about pulling in this whole package (the minified bundle is 86k if I [download it](https://cdnjs.com/libraries/proj4js) by itself and its architecture doesn't look amenable to tree shaking) just to get one presumably very simple pair of converters. We can move forward now but let's investigate whether there's a simpler way to do this in the future.